front/lib/api/assistant/generation.ts

```diff
@@ -277,35 +285,47 @@ export async function constructPrompt(
   fallbackPrompt?: string
 ) {
   const d = moment(new Date()).tz(userMessage.context.timezone);
+  const owner = auth.workspace();
 
-  let meta = "";
-  meta += `ASSISTANT: @${configuration.name}\n`;
-  meta += `LOCAL_TIME: ${d.format("YYYY-MM-DD HH:mm (ddd)")}\n`;
+  let context = "CONTEXT:\n";
+  context += `{\n`;
+  context += `  "assistant": "@${configuration.name}",\n`;
+  context += `  "local_time": "${d.format("YYYY-MM-DD HH:mm (ddd)")}",\n`;
+  if (owner) {
+    context += `  "workspace": "${owner.name}",\n`;
+  }
+  context += "}\n";
+
+  let instructions = "INSTRUCTIONS:\n";
```

Comment:
Reading the code below, it looks like we could end up calling the model with `INSTRUCTIONS:` not followed by any proper instructions, is it okay?